src/core/processing/qgsprocessingparameters.cpp

```diff
@@ -5363,7 +5364,17 @@ QString QgsProcessingParameterExpression::asPythonString( const QgsProcessing::P
       code += QStringLiteral( ", parentLayerParameterName='%1'" ).arg( mParentLayerParameterName );
 
       QgsProcessingContext c;
-      code += QStringLiteral( ", defaultValue=%1)" ).arg( valueAsPythonString( mDefault, c ) );
+      code += QStringLiteral( ", defaultValue=%1" ).arg( valueAsPythonString( mDefault, c ) );
+
+      switch ( mExpressionType )
+      {
+        case Qgis:
+          code += QStringLiteral( ", type=QgsProcessingParameterExpression.Qgis)" );
```

Comment:
Given this is the default anyway, let's omit the argument so that there's a bit more compatibility with the python command with older QGIS releases.

Reply:
Done